pages/en/android-environment/deploying-apps.md

```diff
@@ -91,16 +95,47 @@ adb install [path to your APK]
 
 If you can't use the method described above and need to push your app from another device, you can connect the device to ADB using [USB](#connect-to-adb-over-usb) or a [network address](#connect-to-adb-over-a-network).
 
-To push your APK from another device into the Chromebook, you must start your Chrome OS in [developer mode](https://chromium.googlesource.com/chromiumos/docs/+/master/developer_mode.md) so that you can configure the Chromebook and push apps from the host machine. Follow this steps to get into [developer mode](/{{locale.code}}/productivity/experimental-features#developer-mode).
+### Connect to ADB over a network {: #adb-ip}
+
+1. Make sure you [enabled ADB debugging.](#enable-adb-debug)
+
+Get the IP address of your Chromebook:
+
+1.  Click the clock in the bottom-right area of the screen.
```

Comment:
There's a faster way to do this. Clock > Wifi icon > Info (that i in a circle). This will produce a tooltip-style dialog that shows the IP address, and it won't open a new program.